.github/workflows/bluetooth-tests.yaml

```diff
@@ -41,7 +41,13 @@ jobs:
         uses: actions/checkout@v2
```

Comment:
This does shallow clone and rebasing onto a different branch will not work.

> fatal: invalid upstream 'origin/main'

Need to specify `fetch-depth: 0`.